returns/io.py

```diff
@@ -604,7 +606,7 @@ def failure(self) -> IO[_ErrorType]:
             ...
           returns.primitives.exceptions.UnwrapFailedError
 
-        """
+        """  # noqa: RST399
```

Comment:
Why do we need this `noqa`?

Reply:
It's because we need add the `:force:` directive, without this directive `pygments` throw a warning!!
`flake8` was warning because that.

```python
class Immutable(object):
    """
    Helper type for objects that should be immutable.

    When applied, each instance becomes immutable.
    Nothing can be added or deleted from it.

    .. code:: pycon
      :force:  # HERE

      >>> from returns.primitives.types import Immutable
      >>> class MyModel(Immutable):
      ...     ...

      >>> model = MyModel()
      >>> model.prop = 1
      Traceback (most recent call last):
         ...
      returns.primitives.exceptions.ImmutableStateError

    See :class:`returns.primitives.container.BaseContainer` for examples.

    """
```

flake8 output:
```text
  579:1    RST399 Error in "code" directive:
  """
        Get value from successful container or raise exception for failed one.

        .. code:: pycon
          :force:

          >>> from returns.io import IO, IOFailure, IOSuccess
          >>> assert IOSuccess(1).unwrap() == IO(1)

          >>> IOFailure(1).unwrap()
          Traceback (most recent call last):
            ...
          returns.primitives.exceptions.UnwrapFailedError

        """
  ^
```

Reply:
Can you please report it in `flake8-rst-docstrings`? This is not correct.

Reply:
I'll open the issue, but we'll can remove the `:force:` after the `pygments 2.7.2` asap!
https://github.com/pygments/pygments/issues/1548

Reply:
https://github.com/peterjc/flake8-rst-docstrings/issues/28